### PR TITLE
Add Japanese translation

### PR DESCRIPTION
**Pull request by Dirain1700**
<b>Some contents have been translated.</b>
`pages/ladderhelp`, `pages/rng`, `pages/staff` was replaced to link of Google Docs.
There are written by ililililililili, he is a Room Moderator in Japanese and there are also written in RoomFAQs of Japanese.

### CODE REVIEW

File: translations/japanese/main.ts [hunk 1]
```diff
@@ -118,52 +118,52 @@ export const translations: Translations = {
...
-		"You are ${lockType} and can only private message members of the global moderation team. ${lockExpiration}": "",
-		"Get help with this": "",
-		"The user \"${targetUser.name}\" is locked and cannot be PMed.": "",
-		"On this server, you must be of rank ${groupName} or higher to PM users.": "",
-		"This user is blocking private messages right now.": "",
-		"This ${Config.groups[targetUser.group].name} is too busy to answer private messages right now. Please contact a different staff member.": "",
-		"If you need help, try opening a <a href=\"view-help-request\" class=\"button\">help ticket</a>": "",
-		"You are blocking private messages right now.": "",
-		"You are blocking challenges right now.": "",
+
```
Comment (Zarel) [inline translations/japanese/main.ts:121]:
```suggestion

```

File: translations/japanese/main.ts [hunk 2]
```diff
@@ -118,52 +118,52 @@ export const translations: Translations = {
...
-		"You are now blocking challenges, except from staff and ${rank}.": "",
-		"You are now blocking challenges, except from staff and ${status} users.": "",
-
-		"Staff FAQ": "",
-		"You cannot broadcast all FAQs at once.": "",
-		"A user is autoconfirmed when they have won at least one rated battle and have been registered for one week or longer. In order to prevent spamming and trolling, most chatrooms only allow autoconfirmed users to chat. If you are not autoconfirmed, you can politely PM a staff member (staff have %, @, or # in front of their username) in the room you would like to chat and ask them to disable modchat. However, staff are not obligated to disable modchat.": "",
-		"How the ladder works": "",
+		"You are now blocking challenges, except from staff and ${rank}.": "対戦の申し込みをブロックしました。グローバルスタッフと${rank}権限を持つユーザーからはブロックされません。",
+		"You are now blocking challenges, except from staff and ${status} users.": "対戦の申し込みをブロックしました。グローバルスタッフと${status}状態のユーザーからはブロックされません。",
+
```
Comment (Zarel) [inline translations/japanese/main.ts:148]:
```suggestion

```

File: translations/japanese/main.ts [hunk 3]
```diff
@@ -118,52 +118,52 @@ export const translations: Translations = {
...
+		"To join a room tournament, click the <strong>Join!</strong> button or type the command <code>/tour join</code> in the room's chat. You can check if your team is legal for the tournament by clicking the <strong>Validate</strong> button once you've joined and selected a team. To battle your opponent in the tournament, click the <strong>Ready!</strong> button when it appears. There are two different types of room tournaments: elimination (if a user loses more than a certain number of times, they are eliminated) and round robin (all users play against each other, and the user with the most wins is the winner).": "チャットルームのトーナメントに参加するには、<strong>Join!</strong>のボタンをクリックするか<code>/tour join</code>を部屋のチャットで送信します。<strong>Validate</strong>をクリックすることによって選択したチームがトーナメントのルールに適合しているか確認することができます。<strong>Ready!</strong>ボタンが表示されたときにそれをクリックすると、トーナメントの対戦相手とバトルを開始します。また、2つの形式のトーナメントが存在します。Elimination(トーナメント形式。規定された回数を負け越すと強制的にトーナメントから敗退になります。)とRound Robin(総当たり。トーナメントの参加者全員とバトルをします。勝った回数が1番多いユーザーが優勝になります。)",
+		"Frequently Asked Questions": "よくある質問",
+
+		"pages/faq": "pages/faq-ja",
+		"pages/ladderhelp": "pages/ladderhelp-ja",
+		"pages/rng": "pages/rng-ja",
+		"pages/staff": "pages/staff-ja",
+
```
Comment (Zarel) [inline translations/japanese/main.ts:163]:
```suggestion

```

File: translations/japanese/main.ts [hunk 4]
```diff
@@ -118,52 +118,52 @@ export const translations: Translations = {
...
+		"pages/ladderhelp": "interstice?uri=https%3A%2F%2Fdocs.google.com%2Fdocument%2Fd%2Fe%2F2PACX-1vTww62_gyYb8SRqGATPfIyOBMs7QYfrGWgVPGlB_binqi_XwnC8Q-mNPwC6tyDOp2t1EpgB1bNJOQMB%2Fpub",
+		"pages/rng": "interstice?uri=https%3A%2F%2Fdocs.google.com%2Fdocument%2Fd%2Fe%2F2PACX-1vRMQCbc2iVw1fooQ1N3kKoRZAjDa4uZLIe3HzJ0EQyC6_vxSd6iM0XrIkem1XlwPZy39fJ4gE59wXRM%2Fpub",
+		"pages/staff": "interstice?uri=https%3A%2F%2Fdocs.google.com%2Fdocument%2Fd%2Fe%2F2PACX-1vQcpqSiE3npJs1LJzR1jTvS2JTULPlLegg9Ry2aa0HJi5-A4lottAsxktOOcbyzcZrrcix415karrbM%2Fpub",
+		
+		"- We log PMs so you can report them - staff can't look at them without permission unless there's a law enforcement reason.": "PMはユーザーからの報告などに備えて記録されていますが、罰の執行など正当な理由がない限り閲覧することはできません。",
+		"- We log IPs to enforce bans and mutes.": "IPアドレスはBANやミュートのために記録されています。",
+		"- We use cookies to save your login info and teams, and for Google Analytics and AdSense.": "ログイン情報やチームの保存、Googleアナリティクスやアドセンスのためにクッキーが使用されています。",
+		'- For more information, you can read our <a href="https://${Config.routes.root}/privacy">full privacy policy.</a>': 'もっと知りたい場合は、<a href="https://${Config.routes.root}/privacy">をお読みください。',
```
[the remark this answers is not in the document]
Comment (Zarel) [reply translations/japanese/main.ts:167]: You marked this as resolved but this doesn't seem to have been fixed yet.

**Comment by Dirain1700** [comment]
oh, thank you Zarel

**Comment by Zarel** [comment]
Your `</a>` issue still hasn't been fixed.

**Comment by Dirain1700** [comment]
I want to edit other files besides main.ts so I will set this PR to Draft.
[smogon/pokemon-showdown/translations/english](https://github.com/smogon/pokemon-showdown/tree/master/translations/english)

**Comment by Dirain1700** [comment]
Close this PR with only changes to [main.ts](https://github.com/Dirain1700/pokemon-showdown/blob/patch-1/translations/japanese/main.ts), as the translation of the other files requires more time.
The other files will be merged into a separate PR.

I'm ready to merge!

File: translations/japanese/main.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (Dirain1700) [reply translations/japanese/main.ts]: Sorry, I fixed it now.

**Comment by Zarel** [comment]
Thank you!

**Pull request merged**